src/test/scala/io/iohk/ethereum/network/discovery/PeerDiscoveryManagerSpec.scala

```diff
@@ -44,7 +44,7 @@ class PeerDiscoveryManagerSpec extends FlatSpec with Matchers with MockFactory w
 
     val expectedFindNodeResponse = SendMessage(FindNode(ByteString(nodeStatus.nodeId), expectedTime), remoteUdpAddress)
 
-    Thread.sleep(500)
+    Thread.sleep(1500)
```

Comment:
Why this change was introduced in a `README` update PR?

Reply:
Is this still necessary?

Reply:
Cli release is a totally different branch from daeadalus release branch. This timeout increase was introduced already in `phase/daeadlus` release and in `phase/release1_1` so to avoid spurious tests fails it made sense to introduce it here.